www/pages/routes-and-stops/routes-and-stops-controller.js

```diff
@@ -8,43 +8,58 @@ angular.module('pvta.controllers').controller('RoutesAndStopsController', functi
   /*
    * Get all the routes and stops
    */
+
+   $scope.redirectRoute = function(routeId){
```

Comment:
Comment this and [this](https://github.com/umts/pvta-multiplatform/pull/268/files?diff=unified#diff-365f07089626b424259d271203b2ff1eR13).